services/keybase/keybase-btc.service.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+const KeybaseProfile = require('./keybase-profile')
+const Joi = require('joi')
+
+const bitcoinAddressFoundSchema = Joi.object({
+  them: Joi.array()
+    .items(
+      Joi.object({
+        cryptocurrency_addresses: {
+          bitcoin: Joi.array()
+            .items(
+              Joi.object({
+                address: Joi.string().required(),
+              }).required()
+            )
+            .required(),
+        },
+      }).required()
+    )
+    .min(0)
+    .max(1),
+}).required()
+
+const profileNotFoundSchema = Joi.object({
+  them: Joi.array().empty(),
+}).required()
+
+const bitcoinAddressFoundOrNotFound = Joi.alternatives(
+  bitcoinAddressFoundSchema,
+  profileNotFoundSchema
+)
+
+module.exports = class KeybaseBTC extends KeybaseProfile {
+  static get apiVersion() {
+    return '1.0'
+  }
+
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'keybase/btc',
+      pattern: ':username',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'btc' }
+  }
+
+  async handle({ username }) {
+    const options = {
+      method: 'GET',
```

Comment:
it's not necessary to include the method here unless you need to use a verb other than `get`

Reply:
Good to know. Thanks.